consensus/istanbul/validator/doc.go

```diff
@@ -14,5 +14,34 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Package validator implements the types related to the validators participating in istanbul consensus.
+/*
+Package validator implements the types related to the validators participating in istanbul consensus.
+
+Types in validator package implement `Validator` and `ValidatorSet` interface in istanbul/validator.go file.
+Those types are used for validating blocks to make consensus.
+
+Validator
+
+`Validator` is a node which has 2 features for consensus.
+First, it has a right to make a block. Only validator can make a block.
+Second, it checks blocks made by other validators are valid or not. 2/3 of validators should approve for
```

Comment:
```suggestion
Validate: A validator node can validate blocks from proposers. A block is validated only if more than 2/3 of validators approve the given block.
```

Reply:
"2/3 of validators should approve for Klaytn uses weightedValidator for Klaytn consensus" sounds a bit awkward. Could you please check the last sentence?

Reply:
reflected